### PR TITLE
DEV-1477: Update Transfer Processing times article

### PR DESCRIPTION
**Pull request by ShreyaThapa**

### CODE REVIEW

File: pages/docs/balance/transfer-processing-times.mdx [hunk 1]
```diff
@@ -5,138 +5,243 @@ description: Learn more about transfer clearing times, including `standard` ,`ex
...
+A Standard ACH debit from a bank to the Dwolla Network processes within 3-4 business days. This extended processing time accounts for ACH returns and processing delays, and checks on [common return codes](https://www.dwolla.com/resources/ach-return-codes/) like insufficient funds and invalid bank accounts. If a failure occurs during ACH processing, the transfer will be marked with a return code. For detailed information on ACH return codes and [transfer failures](https://developers.dwolla.com/concepts/transfer-failures), refer to our developer resource article.
```
Comment (jcoon97) [inline pages/docs/balance/transfer-processing-times.mdx]:
```suggestion
A Standard ACH debit from a bank to the Dwolla Network processes within 3-4 business days. This extended processing time accounts for ACH returns and processing delays, and checks on [common return codes](https://www.dwolla.com/resources/ach-return-codes/) like insufficient funds and invalid bank accounts. If a failure occurs during ACH processing, the transfer will be marked with a return code. For detailed information on ACH return codes and [transfer failures](/docs/balance/transfer-failures), refer to our developer resource article.
```

Comment (jcoon97) [reply pages/docs/balance/transfer-processing-times.mdx]: Updated URI to relative using `/docs/balance`

File: pages/docs/balance/transfer-processing-times.mdx [hunk 2]
```diff
@@ -5,138 +5,243 @@ description: Learn more about transfer clearing times, including `standard` ,`ex
...
+| Monday 4 p.m. to Tuesday 4 p.m.     | Tuesday 4 p.m.    | Wednesday morning  |
+| Tuesday 4 p.m. to Wednesday 4 p.m.  | Wednesday 4 p.m.  | Thursday morning   |
+| Wednesday 4 p.m. to Thursday 4 p.m. | Thursday 4 p.m.   | Friday morning     |
+| Thursday 4 p.m. to Friday 4 p.m.    | Friday 4 p.m.     | Monday morning     |
+
+_\*Dwolla cannot guarantee an exact deadline for transmission of ACH files. Please be mindful that these are target cut-off times for transfer processing._
+
+## Same Day ACH Transfers
+
+We can enable [Same Day ACH](https://developers.dwolla.com/concepts/same-day-ach) transfers on an account-level basis for transfers within your application. With Same Day ACH, you can apply it to the ACH debit, the ACH credit, or both. After receiving approval from Dwolla to enable Same Day ACH for your account, you can create a Same Day debit and/or credit on a per-transfer basis by specifying it in the clearing object of an API transfer request.
```
Comment (jcoon97) [inline pages/docs/balance/transfer-processing-times.mdx]:
```suggestion
We can enable [Same Day ACH](/docs/balance/same-day-ach) transfers on an account-level basis for transfers within your application. With Same Day ACH, you can apply it to the ACH debit, the ACH credit, or both. After receiving approval from Dwolla to enable Same Day ACH for your account, you can create a Same Day debit and/or credit on a per-transfer basis by specifying it in the clearing object of an API transfer request.
```

Comment (jcoon97) [reply pages/docs/balance/transfer-processing-times.mdx]: Same as above :)

File: pages/docs/balance/transfer-processing-times.mdx [hunk 3]
```diff
@@ -5,138 +5,243 @@ description: Learn more about transfer clearing times, including `standard` ,`ex
...
+- 1 p.m. CT
+- 3 p.m. CT\*
+  \*Note: These may vary depending on the financial institution.
+- Transfer processing time:
+- Debit: 0-1 business day
+- Credit: 0-1 business day
+- Processed on business days (processing unavailable on weekends and banking holidays)
+- T[ransaction limit: $1 million](https://www.dwolla.com/updates/access-faster-payments-with-dwolla-same-day-ach/) (enforced by Nacha)
+
+### Same Day ACH Debit ( Payment In)
```
Comment (jcoon97) [inline pages/docs/balance/transfer-processing-times.mdx]:
```suggestion
### Same Day ACH Debit (Payment In)
```

File: pages/docs/balance/transfer-processing-times.mdx [hunk 4]
```diff
@@ -5,138 +5,243 @@ description: Learn more about transfer clearing times, including `standard` ,`ex
...
+| ------------------------ | ----------------- | ------------------ |
+| 3 p.m. to 9 a.m.         | 9 a.m.            | 11:30 a.m.         |
+| 9 a.m. to 1 p.m.         | 1 p.m.            | 5 p.m.             |
+| 1 p.m. to 3 p.m.         | 3 p.m.            | 5 p.m.             |
 
-### Clearing example
+_\*Dwolla cannot guarantee an exact deadline for transmission of ACH files. Please be mindful that these are target cut-off times for transfer processing._
 
-In this bank-to-bank transfer request example we have `next-day` ACH enabled, but we want to slow the transfer clearing into the Dwolla Network back to `standard` ACH. However, we feel comfortable letting this transfer export out of the Dwolla Network with `same-day` ACH clearing to the destination bank.
+### Same Day ACH Credit ( Payment Out)
```
Comment (jcoon97) [inline pages/docs/balance/transfer-processing-times.mdx]:
```suggestion
### Same Day ACH Credit (Payment Out)
```

File: pages/docs/balance/transfer-processing-times.mdx [hunk 5]
```diff
@@ -5,138 +5,243 @@ description: Learn more about transfer clearing times, including `standard` ,`ex
...
+| ------------------------ | ----------------- | ------------------ |
+| 3 p.m. to 9 a.m.         | 9 a.m.            | 11:30 a.m.         |
+| 9 a.m. to 1 p.m.         | 1 p.m.            | 5 p.m.             |
+| 1 p.m. to 3 p.m.         | 3 p.m.            | 5 p.m.             |
+
+_\*Dwolla cannot guarantee an exact deadline for transmission of ACH files. Please be mindful that these are target cut-off times for transfer processing._
+
+## RTP (Real-Time Payments)
+
+[RTP](https://developers.dwolla.com/concepts/real-time-payments) is a payment method that allows for near-instantaneous transfer processing.
```
Comment (jcoon97) [inline pages/docs/balance/transfer-processing-times.mdx]:
```suggestion
[RTP](/docs/balance/real-time-payments) is a payment method that allows for near-instantaneous transfer processing.
```

File: pages/docs/balance/transfer-processing-times.mdx [hunk 6]
```diff
@@ -5,138 +5,243 @@ description: Learn more about transfer clearing times, including `standard` ,`ex
...
+
+Key Points to Note for RTP Transfers:
+
+- Available for credit transfers only
+- Export deadline: None
+- Transfer processing time: Funds available within minutes
+- Processed 24/7/365 ( business days, weekends and banking holidays)
```
Comment (jcoon97) [inline pages/docs/balance/transfer-processing-times.mdx]:
```suggestion
- Processed 24/7/365 (business days, weekends and banking holidays)
```

File: pages/docs/balance/transfer-processing-times.mdx [hunk 7]
```diff
@@ -5,138 +5,243 @@ description: Learn more about transfer clearing times, including `standard` ,`ex
...
+- Export deadline: Multiple throughout the day:
+- 9 a.m. CT
+- 1 p.m. CT
+- 3 p.m. CT\*
+  \*Note: These may vary depending on the financial institution.
+- Transfer processing time:
+- Debit: 0-1 business day
+- Credit: 0-1 business day
+- Processed on business days (processing unavailable on weekends and banking holidays)
+- T[ransaction limit: $1 million](https://www.dwolla.com/updates/access-faster-payments-with-dwolla-same-day-ach/) (enforced by Nacha)
```
Comment (jcoon97) [inline pages/docs/balance/transfer-processing-times.mdx]:
```suggestion
- [Transaction limit: $1 million](https://www.dwolla.com/updates/access-faster-payments-with-dwolla-same-day-ach/) (enforced by Nacha)
```

**Comment by ShreyaThapa** [comment]
Thanks for catching those wrong links and spacing errors, @jcoon97! 🙌 
Accepted all suggestions and re-requested review 😃

**Review by jcoon97 — APPROVED** [review]

**Pull request merged**